src/main/java/org/assertj/guava/error/RangeSetShouldIntersectAnyOf.java

```diff
@@ -0,0 +1,233 @@
+ *       and charge a fee for, acceptance of support, warranty, indemnity,
+ *       or other liability obligations and/or rights consistent with this
+ *       License. However, in accepting such obligations, You may act only
+ *       on Your own behalf and on Your sole responsibility, not on behalf
+ *       of any other Contributor, and only if You agree to indemnify,
+ *       defend, and hold each Contributor harmless for any liability
+ *       incurred by, or claims asserted against, such Contributor by reason
+ *       of your accepting any such warranty or additional liability.
+ *
+ *    END OF TERMS AND CONDITIONS
+ *
+ *    APPENDIX: How to apply the Apache License to your work.
+ *
+ *       To apply the Apache License to your work, attach the following
+ *       boilerplate notice, with the fields enclosed by brackets "[]"
+ *       replaced with your own identifying information. (Don't include
+ *       the brackets!)  The text should be enclosed in the appropriate
+ *       comment syntax for the file format. We also recommend that a
+ *       file or class name and description of purpose be included on the
+ *       same "printed page" as the copyright notice for easier
+ *       identification within third-party archives.
+ *
+ *    Copyright [yyyy] [name of copyright owner]
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package org.assertj.guava.error;
+
+import org.assertj.core.error.BasicErrorMessageFactory;
+import org.assertj.core.error.ErrorMessageFactory;
+
+import com.google.common.collect.RangeSet;
+
+/**
+ * Creates an error message indicating that the given {@link com.google.common.collect.RangeSet} does not intersect 
+ * at lease one element of expected objects.
+ * 
+ * @author Ilya Koshaleu
+ */
+public class RangeSetShouldIntersectAnyOf extends BasicErrorMessageFactory {
+
+  public static ErrorMessageFactory shouldIntersectAnyOf(RangeSet actual, Object expected) {
+    return new RangeSetShouldIntersectAnyOf(actual, expected);
+  }
+
+  /**
+   * Creates a new <code>{@link BasicErrorMessageFactory}</code>.
+   *
+   * @param actual actual {@link com.google.common.collect.RangeSet}.
+   * @param expected expected range to intersect.
+   */
+  private RangeSetShouldIntersectAnyOf(Object actual, Object expected) {
+    super("%nExpecting%n <%s>%nshould intersect at least one range of the given:%n <%s>%n", actual, expected);
```

Comment:
wording: `intersect at least one range of the given:` -> `intersect at least one range in:`

Reply:
fixed